scripts/1.download.py

```diff
@@ -0,0 +1,52 @@
+
+# coding: utf-8
```

Comment:
I think this is an important script and should be the number 1 file in the repo - but I did like the fact that both example scripts could run independently. It looks like with this update a user will need to first run the download script